client/packages/coldchain/src/Monitoring/ListView/TemperatureChart/useTemperatureChartData.ts

```diff
@@ -6,14 +7,57 @@ import {
   TemperatureChartFragment,
   useTemperatureChart,
 } from '../../api/TemperatureChart';
-import { TemperatureLogFilterInput } from '@common/types';
+import {
+  DatetimeFilterInput,
+  InputMaybe,
+  TemperatureLogFilterInput,
+} from '@common/types';
 
 const MAX_DATA_POINTS = 30;
 const BREACH_RANGE = 2;
 const BREACH_MIN = 2;
 const BREACH_MAX = 8;
 
+const useFilterDates = (
+  filterDatetime: InputMaybe<DatetimeFilterInput> | undefined
+) => {
+  const now = DateUtils.setMilliseconds(new Date(), 0);
```

Comment:
this is mostly just a move - I lifted it out so that I could experiment with memoising and decided to leave it, it's neater in a separate function